OpenRA.Game/Effects/FlashTarget.cs

```diff
@@ -25,7 +25,7 @@ public FlashTarget(Actor target, Player asPlayer = null, int ticks = 4)
 			this.target = target;
 			player = asPlayer;
 			remainingTicks = ticks;
-			foreach (var e in target.World.Effects.OfType<FlashTarget>().Where(a => a.target == target).ToArray())
+			foreach (var e in target.World.Effects.OfType<FlashTarget>().Where(a => a.target == target).ToList())
```

Comment:
In this case, because the collection size is unknown, ToList is very slightly faster than ToArray.